tests/test_clients.py

```diff
@@ -0,0 +1,298 @@
+import datetime
+import os
+import time
+import uuid
+from typing import Optional, Union
+
+import pytest
+import redis
+from _pytest.fixtures import FixtureRequest
+
+from momento_python_redis_client.momento_redis_client import MomentoRedis
+
+TClient = Union[MomentoRedis, redis.Redis]
+
+skip_redis = os.getenv("TEST_SKIP_REDIS", False)
+skip_momento = os.getenv("TEST_SKIP_MOMENTO", False)
+
+
+def _use_client(client: str) -> bool:
+    if client == "momento_redis_client" and skip_momento:
+        return False
+    elif client == "redis_client" and skip_redis:
+        return False
+    return True
```

Comment:
Took me a minute to understand the default was "run both" we can independently opt out of each one. I think the code below may be more maintainable if we have one environment variable to toggle between the momento v redis; the tests only use one client (can get rid of the if tests and parametrized thing); then run the tests twice.

Not a blocker but something to consider for a future PR

Reply:
We can consider changing these up, but I really like having the ability to run them both at the same time so we can easily and simultaneously verify both the internal consistency of each library as well as the consistency of results between them without having to collate results from multiple runs. I'm going to leave them as they are for now but am totally open to having a conversation about improving this in the future.